pkg/crd/markers/doc.go

```diff
@@ -26,20 +26,32 @@ limitations under the License.
 // be run after the rest of a given schema node has been generated.
 // Markers that need to be run before any other markers can also
 // implement ApplyFirst, but this is discouraged and may change
-// in the future.
+// in the future. It is recommended to implement the ApplyPriority
+// interface in combination with ApplyPriorityDefault and
+// ApplyPriorityFirst constants.
 //
+//  type MyCustomMarker string
+//
+//  func (m MyCustomMarker) ApplyPriority() ApplyPriority {
+//  	return ApplyPriorityFirst
+//  }
+//
+//  func (m MyCustomMarker) ApplyToSchema(schema *apiext.JSONSchemaProps) error {
+//    ...
+//  }
```

Comment:
Could we change it for. something like

interface in combination with ApplyPriorityDefault and
// ApplyPriorityFirst constants. Following an example over how to use them:

Reply:
Done